bot/controllers/twitchLive.js

```diff
@@ -1,6 +1,6 @@
 const BaseController = require('../baseController.js');
 const Command = require('../baseCommand.js');
-const util = require('apex-util');
+// const util = require('apex-util');
 
 class StreamerLiveController extends BaseController {
```

Comment:
Would it be more appropriate to name the class in tangent with the file name? Example: TwitchLiveController